### PR TITLE
Timezone-aware MongoDB

### LINKED ISSUE #231: Make MongoDB database handler timezone-aware
To preserve the timezone-aware attribute in datetime object, it is necessary to use the `tz_aware=True` option in MongoDBClient.

More info:
https://pymongo.readthedocs.io/en/stable/examples/datetimes.html

### PR DESCRIPTION
**Pull request by byewokko**
This ensures that datetime objects retrieved from MongoDB contain UTC timezone info.

Closes #231

### CODE REVIEW

File: asab/storage/mongodb.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		'asab:storage': {
 			'mongodb_uri': 'mongodb://localhost:27017',
 			'mongodb_database': 'asabdb',
+			'timezone_aware': 'yes',
```
Comment (ateska) [inline asab/storage/mongodb.py:15]: That should not be configurable - it will lead to diverse and complicated ecosystem.

**Review by ateska — CHANGES_REQUESTED** [review]

File: asab/storage/mongodb.py [hunk 2]
```diff
@@ -26,7 +26,12 @@ class StorageService(StorageServiceABC):
 	def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Client = motor.motor_asyncio.AsyncIOMotorClient(asab.Config.get(config_section_name, 'mongodb_uri'))
-		self.Database = self.Client[asab.Config.get(config_section_name, 'mongodb_database')]
+
+		self.Database = self.Client.get_database(
+			asab.Config.get(config_section_name, 'mongodb_database'),
+			codec_options=bson.codec_options.CodecOptions(tz_aware=True),
```
Comment (ateska) [inline asab/storage/mongodb.py:32]: `tz_aware` - it really means that the datetime is in UTC? Not local time ...

**Review by ateska — CHANGES_REQUESTED** [review]
One question, then mergeable.

File: asab/storage/mongodb.py (hunk 2 above)
Comment (ateska) [reply asab/storage/mongodb.py:32]:
```
tz_aware=True,
...     tzinfo=timezone.utc))
```

?

Comment (byewokko) [reply asab/storage/mongodb.py:32]: Okay, I just read more on this so here is a clarification:

- There is **no way** to preserve the `tzinfo` attribute in datetime objects in Mongo.
- `tz_aware=True` only causes the **python client** to add `tzinfo` to all datetime objects retrieved from the database. Inside the database they are still timezone-naive.
- If no `tzinfo` parameter is specified, the retrieved objects get `tzinfo=timezone.utc` as default.

**Pull request merged**